source/_includes/common-tasks/define_custom_polling.md

```diff
@@ -1,6 +1,13 @@
 
 
-If you want to define a specific interval at which your device is being polled for data, you can disable the default polling interval and create your own polling service.
+If you want to define a specific interval at which your device is being polled for data, you can disable the default polling interval and create your own polling automation.
+
+The reason why we chose this approach is because this gives you way more flexibility on when to poll.
+
+1. This already works with most of the integrations. Since changing polling interval is something that could be a use case for ever integration, it makes sense to pick an approach that works for every integration without having to add support for every individual integrations.
```

Comment:
```suggestion
1. This already works with most of the integrations. Since changing the polling interval could be a use case for every integration, it makes sense to pick an approach that works for every integration without having to add support to every individual integration.
```